src/parser/jobs/rpr/modules/Reaver.tsx

```diff
@@ -0,0 +1,137 @@
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const SEVERITY_TIERS = {
+	1: SEVERITY.MEDIUM,
+	2: SEVERITY.MAJOR,
+}
+
+const SOUL_GAIN = 1 //Soul moves grant 1 reaver
+const GLUTTONY_GAIN = 2 //Gluttony grants 2 reavers, also therotically max
+
+export class Reaver extends Analyser {
+	static override handle = 'reaver'
+
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+
+	//Trackers
+	private haveReaver = false
```

Comment:
Is tracking this independently useful, or could you just `this.actors.current.hasStatus(this.data.statuses.SOUL_REAVER.id)`?